src/index.ts

```diff
@@ -101,28 +127,40 @@ async function run(): Promise<void> {
 
     // send the deployment
     const deployment: Deployment = await apiClient.postDeploymentEvent(
-      applicationId,
-      environmentId,
       browserTypes,
-      uri,
+      planLabels,
+      httpHeaders,
       rebaselineImages,
       setStaticBaseline,
-      revision,
       eventTime,
       properties,
+      applicationId,
+      environmentId,
+      uri,
+      revision,
+      mablBranch,
     );
 
-    core.setOutput('mabl-deployment-id', deployment.id);
+    core.setOutput(ActionOutputs.DeploymentId, deployment.id);
 
-    let outputLink: string = baseApiUrl;
+    let appOrEnv: Application | Environment | undefined;
```

Comment:
Fail out if neither app-id or env-id are valid and supplied.

Reply:
It looks like as long as the application ID is valid, the environment ID does not get checked. Does the Deployment entity contain info about the number of plans/tests that are going to be started?

Reply:
Thanks, @simonychoy . This validation pass is confirming that we have at least one valid entity, to build the UI app URL. The backing REST API will conduct a more thorough check.